extensions/ql-vscode/src/queries-panel/query-pack-discovery.ts

```diff
@@ -71,32 +67,13 @@ export class QueryPackDiscovery extends FilePathDiscovery<QueryPack> {
   }
 
   protected async getDataForPath(path: string): Promise<QueryPack> {
-    const language = await this.determinePackLanguage(path);
-    return { path, language };
-  }
-
-  private async determinePackLanguage(
-    path: string,
-  ): Promise<QueryLanguage | undefined> {
-    let packInfo: QuerySetup | undefined = undefined;
+    let language: QueryLanguage | undefined;
     try {
-      packInfo = await this.cliServer.resolveLibraryPath(
-        getOnDiskWorkspaceFolders(),
-        path,
-        true,
-      );
-    } catch (err) {
-      void extLogger.log(
-        `Query pack discovery failed to determine language for query pack: ${path}${EOL}Reason: ${getErrorMessage(
-          err,
-        )}`,
-      );
+      language = await getQlPackLanguage(path);
+    } catch (e) {
```

Comment:
We should probably log this exception. It won't be as noisy as it was before, but we should still highlight if this code does fail.

I think the only reasons for an error here are the file not existing (which shouldn't happen unless the file was just deleted) or there's a programming bug in `getQlPackLanguage`.

Side note: I'm surprised this isn't giving a warning for an unused variable.